### DISCUSSION TITLE
Is it possible to change the number of songs that can be queued from a playlist?

### DISCUSSION THREAD

**Discussion by FordFriedel** [Androz2091/discord-player#1105] [Q&A] [answered]
When queuing a (spotify) playlist it appears as if you can only get the first 100 tracks at a time, is it possible to change this?

**Comment by twlite** [comment, marked as answer]
It is not possible with the default implementation as spotify only gives first 100 tracks from the widget. You can however use the spotify api and parse all tracks then load it into the player to get more than 100 tracks.

**Reply by FordFriedel** [reply]
Thanks!
